hamocc/mo_read_oafx.F90

```diff
@@ -77,13 +82,19 @@ module mo_read_oafx
   !                surface ocean (assumed to be between 60S and 70N)
   !  const_0p56    Homogeneous addition of 0.56 Pmol ALK/yr-1 over the ice-free
   !                surface ocean (assumed to be between 60S and 70N)
+  !  ramp          Linear increase of homogeneous addition of 0 to 0.135 Pmol
+  !                ALK/yr-1 from 2025 to 2035 over the ice-free surface ocean  
+  !                (assumed to be between 60S and 70N)
   !
   real, parameter :: addalk_0p14   = 0.14  ! Pmol alkalinity/yr added in the
   real, parameter :: addalk_0p56   = 0.56  ! 'const_0p14' and 'const_0p56' 
                                            ! scenarios
   real, parameter :: cdrmip_latmax =  70.0 ! Min and max latitude where
   real, parameter :: cdrmip_latmin = -60.0 ! alkalinity is added according
                                            ! to the CDRMIP protocol
+  real, parameter :: addalk_ramp   = 0.135 ! Max Pmol alkalinity/yr added
+  integer, parameter :: ramp_start = 2025  ! in 2035 in the 'ramp' scenario,
+  integer, parameter :: ramp_end   = 2035  ! starting at 0 Pmol/yr in 2025.
```

Comment:
It is a bit arbitrary to have the ramp-up start and end years as hard-coded parameters. Would it be better to read this from a namelist file? On the other hand, the min/max latitude limits are equally arbitrary ...
@JorgSchwinger , @jmaerz , any thoughts about this?